src/partials/features-zigzag.tsx

```diff
@@ -150,15 +179,25 @@ function FeaturesZigZag() {
                 data-aos="fade-up"
               >
                 <div
-                  className={`max-w-full mx-auto md:max-w-none h-auto flex flex-row flex-wrap fill-[#1c1e21] dark:fill-[#e3e3e3] [color:${
-                    isDarkTheme ? '#e3e3e3' : '#1c1e21'
-                  }]`}
+                  className={`max-w-full mx-auto md:max-w-none h-auto flex flex-row flex-wrap`}
```

Comment:
Same comment as above regarding automating the list. The data can be found [here](https://github.com/open-feature/openfeature.dev/blob/main/src/datasets/sdks/index.ts#L11).

Reply:
I haven't changed this one, I think its better if we use SVG components directly and don't mix with FontAwesomeIcon. Then we could easily refactor and keep it consistent